tests/integration/release/test_runner.py

```diff
@@ -16,13 +16,22 @@
         (["--dataset", "wic", "--year", "2015"]),
         # (["--dataset", "wic", "--population", "USA", "--state", "RI", "--year", "2015"]),
     ],
+    ids=["1", "2", "3", "4"],
```

Comment:
All the changes in this life will eventually be the responsibility of jobmon, so these are just here as a placeholder for testing.